app/uk/gov/hmrc/fhregistrationfrontend/controllers/DeclarationController.scala

```diff
@@ -88,9 +88,16 @@ class DeclarationController @Inject() (
 
   def submitForm() = summaryAction.async { implicit request =>
     val form = declarationForm.bindFromRequest()
+    val errorList = scala.collection.mutable.Set[String]()
     form.fold(
       hasErrors = formWithErrors => {
-        val errors: List[FormError] = formWithErrors.errors.groupBy(_.key).map(x => x._2.head).toList.sortBy(_.key)
+        val errors: Seq[FormError] = formWithErrors.errors.filter { error =>
```

Comment:
`val errors: List[FormError] = formWithErrors.errors.filter { error => !errorList.contains(error.key) }`

Reply:
This doesn't work :( But there is a clearer way of writing this